### PR TITLE
docs: add naifu docs link in readme

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 - 高级请求语法
 - 自定义违禁词表
 - 发送一段时间后自动撤回
-- 连接到私服 · NAIFU · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
+- 连接到私服 · [NAIFU](./docs/deploy/naifu.md) · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
```
Comment (shigma) [inline readme.md:13]: I don't think this should be a relative link. I would prefer <https://bot.novelai.dev/deploy/naifu.md> or <https://rentry.org/sdg_FAQ#naifu-novelai-model-backend-frontend>.

Comment (MaikoTan) [inline readme.md:13]:
```suggestion
- 连接到私服 · [NAIFU](https://bot.novelai.dev/deploy/naifu.md) · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
```

Comment (MaikoTan) [reply readme.md]: Done!

File: readme.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
 - 高级请求语法
 - 自定义违禁词表
 - 发送一段时间后自动撤回
-- 连接到私服 · NAIFU · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
+- 连接到私服 · [NAIFU](https://bot.novelai.dev/deploy/naifu.md) · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
```
Comment (shigma) [inline readme.md:13]:
```suggestion
- 连接到私服 · [NAIFU](https://bot.novelai.dev/deploy/naifu.html) · [SD-WebUI](https://github.com/AUTOMATIC1111/stable-diffusion-webui)
```

**Review by shigma — APPROVED** [review]

**Pull request merged**